angular-spa-ui/js/admin-app/content-manager-module/contentManager-ctrl.js

```diff
@@ -1,90 +1,104 @@
 "use strict";
 
-module.exports = function ($scope, $state, promises, $stateParams, cmService, configService, contentStorage) {
+module.exports = function ($scope, $state, promises, $stateParams, cmService, configService, contentStorage, rlService) {
+
+    var vm = this;
+
+    vm.viewApi = {
+        isActive: function (tabs) {
+            // Set active tab
+            return cmService.isActiveTab(tabs, contentStorage.params.searchIn);
+        },
+        changeTab: function (place) {
+            // Tab click go to state
+            if (contentStorage.params.searchIn == place)
+                return false;
+
+            $state.go(
+                'content', {
+                    searchIn: place
+                }, {
+                    reload: true,
+                    inherit: false
+                }
+            );
+
+        }
+    }
+
+    vm.model = {
```

Comment:
Here vm is model, and vm.model is just property storing data. 
On the view vm = controllersAs name, and we decided to call data container vm.model and methods container vm.viewApi. On the view its looks like search.model and search.viewApi, and its looks logical. So, this is due to the peculiarities of Angular and MVC in angular.
As a variant we can recall vm.model to vm.data